app/eq.py

```diff
@@ -146,12 +145,12 @@ def __init__(self, case: dict, app: Application):
             raise InvalidEqPayLoad(f"Could not retrieve attributes for case {self._case_id}")
 
         try:
-            self._ru_name = self._sample_attributes["Prem1"]
+            self._ru_name = self._sample_attributes["ADDRESS_LINE1"]
```

Comment:
I wonder if this should use the new UPRN value 🤔

Reply:
Sorry bit confused as to what you mean, do you mean removing the country_code and address line for the new UPRN value in the try except?

Reply:
No, I mean checking with eQ to see if they'd rather the "ru_name" (or some other field) use the UPRN for traceability. Before, all we had was the address to identify a respondent but I think UPRN is supposed to be the unique string.

Reply:
Myself and Liam are going to have a discussion with EQ about the country code data we're passing over to them I could bring that up or you could come along tomorrow?

Reply:
Yeah, let's talk about "country code" and the "ru name" tomorrow 👍